src/components/page/AddUserPage.tsx

```diff
@@ -0,0 +1,198 @@
+import { useState } from "react";
+import Sidebar from "../common/Sidebar.tsx";
+import Header from "../common/Header.tsx";
+import Footer from "../common/Footer.tsx";
+import apiClient from "../../api/ApiClient.ts";
+import {Link} from "react-router-dom";
+
+const AddUserPage = () => {
+    const [formData, setFormData] = useState({
+        email: "",
+        password: "",
+        firstName: "",
```

Comment:
It might be worth to ask admin (only one with creation user rights) to repeat the password (?)